### PR TITLE
RB-3 cleaned up UI & added basic search functionality

### PR DESCRIPTION
**Pull request by mapuya19**
- screen clears more often to avoid UI clutter and straightforward navigation
- verbiage is more straightforward for user experience
- basic search implemented (matching substrings)

### CODE REVIEW

File: RecipeBook.java [hunk 1]
```diff
@@ -1,100 +1,281 @@
...
+    System.out.println("-----------------------------------------\n");
   }
 
+  // Display Recipe Info
+  private static void showRecipeInfo(String name) {
+    System.out.println();
+    System.out.println("-----------------------------------------------");
+    System.out.println("Choose how you would like to view the recipe:");
+    System.out.println("0 - View the recipe in its' entirety");
+    System.out.println("1 -  View the recipe's instructions step-by-step");
```
Comment (tangym27) [inline RecipeBook.java]: nit: could you remove the extra space between - and V funder choice 1?

File: RecipeBook.java [hunk 2]
```diff
@@ -1,100 +1,281 @@
...
+          System.out.println("Ingredients: " + x.ingredients);
+          System.out.println();
+          System.out.println("Instructions:");
+          for (String instr : x.instructions) {
+            System.out.println("• " + instr);
+          }
+        }
+      }
+      System.out.println();
+      System.out.print("Press any key to continue: ");
```
Comment (tangym27) [inline RecipeBook.java]: nit: i feel like press is a bit misleading, maybe "enter any key"?

**Review by tangym27 — APPROVED** [review]
great job!

File: RecipeBook.java (hunk 2 above)
Comment (mapuya19) [reply RecipeBook.java]: edited!

File: RecipeBook.java (hunk 1 above)
Comment (mapuya19) [reply RecipeBook.java]: done!

**Pull request merged**